### ISSUE TITLE
SAM_RS：Regarding the reproduction issues of other models in the paper.

### ISSUE THREAD

**Issue by Gcx0707** [sstary/SSRS#53]
For instance, the ABCNet and CMTFNet mentioned in the paper—can you provide the loss functions, training methods, and hyperparameters used? These models' source code does not provide them.

**Comment by sstary** [comment]
Hello,  the training framework for these comparative methods are all the same as our method, which means you can just simply replace the model and train them like our methods. In this github, for the example, in the SAM_RS/utils.py, you can change the model name and train the comparative methods.
